shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.m

```diff
@@ -79,7 +79,6 @@ - (void)setUp {
 }
 
 - (void)tearDown {
-  [engine stopMocking];
```

Comment:
This line caused all the tests in the file to fail. Did we upgrade ocmock or something?

Reply:
Nope we didn't.  You shouldn't remove this line.  The tests are running as part of CI so we know this works before the changes here.  Let me know what errors you are getting and I can try to help.

Reply:
I put it back it starts failing again: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8845604182114825824/+/u/Host_Tests_for_ios_debug_sim/stdout

> /opt/s/w/ir/cache/builder/src/out/ios_debug_sim/../../flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm:1211: error: -[FlutterTextInputPluginTest testPasswordAutofillHack] : failed: caught "NSInternalInconsistencyException", "** Cannot use mock object OCClassMockObject(FlutterEngine) at 0x600002eabb60. This error usually occurs when a mock object is used after stopMocking has been called on it. In most cases it is not necessary to call stopMocking. If you know you have to, please make sure that the mock object is not used afterwards."

Reply:
This means you are leaking an object beyond your test that is talking to the mock engine, probably a view controller.  Let me look at your test code to see if I can spot it.

Reply:
Looks like your test is fine.  One of 2 things are happening:
1) Your test is messing up the timing and surfacing an issue that was already present where we were correctly synchronizing some asynchronous operations (like NSNotifications)
2) You've introduced a leak that is causing view controllers not to be deallocated, which causes them to leak into other tests and break them.

Here's an example where I had to fix similar issues:
https://github.com/flutter/engine/commit/27bf114b861e3bf7a95aef5b90e14e0206d78146#diff-9df3e6878a0f23383e1f2de991386935e435c5b77c863de75bd67a4ced38591dR136

I'd try running that test / writing one similar to make sure that the view controller is being deleted.  If you run that one test and it fails, you'll know you introduced a leak.  Otherwise tracking down the timing issue will take a bit more work.

Reply:
Turns out UIKit is keeping some of the input views alive and modifying the content of these views even they're removed from the view hierarchy. https://github.com/flutter/engine/pull/26547